app/assets/javascripts/discourse/app/components/bookmark-list.gjs

```diff
@@ -358,16 +367,18 @@ export default class BookmarkList extends Component {
                         avatarTemplatePath="avatar_template"
                         usernamePath="username"
                         namePath="name"
-                        imageSize="small"
+                        imageSize="large"
```

Comment:
Not sure if changing this to large is necessary, this takes up a lot of space but I'm not sure why it's needed?

![image](https://github.com/user-attachments/assets/d1e086f0-769b-4e58-a568-927c3dcb0810)